modules/polling/helpers/parseRankedChoiceRawOptionId.ts

```diff
@@ -1,12 +0,0 @@
-import { paddedArray, toBuffer } from 'lib/utils';
-
-export function parseRankedChoiceRawOptionId(rawValue?: string): number[] {
-  let rankedChoiceOption: number[] = [];
-  if (rawValue) {
-    const ballotBuffer = toBuffer(rawValue, { endian: 'little' });
-    const ballot = paddedArray(32 - ballotBuffer.length, ballotBuffer);
-    rankedChoiceOption = ballot.reverse().filter(choice => choice !== 0);
```

Comment:
attention here, we removed the "filter" that removed before all the 0 options, since this assumed 0 is abstain and has to be ignored.